odfuzz/entities.py

```diff
@@ -131,7 +132,51 @@ def build(self):
             query_group_data = QueryGroupData(entity_set, principal_entities, self._restrictions, None)
             self._append_queryable(query_group_data)
 
-        return self._queryable
+        return self._apply_restrictions()
+    
+    def _apply_restrictions(self):
+        #Method for excluding entities and their properties as a part of exclusion_list
+        list_of_entities = self._queryable.all()
```

Comment:
You go trough the _queryable.all() directly in the for loop, no need to declare new variable just for that.